constellations/gemini-admin/src/index.js

```diff
@@ -59,6 +66,17 @@ const SemesterEdit = () => <SemesterEditPage />;
 const SemesterEditMatch = ({ match }) => (
     <SemesterEditPage semesterId={match.params.semesterId} />
 );
+
+const User = () => <UserPage />;
+const UserEdit = () => <UserEditPage />;
+const UserEditMatch = ({ match }) => <UserEditPage id={match.params.id} />;
```

Comment:
just to be more consistent with the rest of the file, you can split this up into two lines (after ({match}) =>)

Reply:
prettify condensed it into one line automatically